tasks/configure-FreeBSD.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: Update hostname in rc.conf(5)
+  lineinfile:
```

Comment:
the last line of `hostname=` in rc.conf is effective and must be replaced with `regexp:` directive.